code/modules/cargo/packs.dm

```diff
@@ -2406,6 +2406,24 @@
 					/obj/item/reagent_containers/food/snacks/grown/cucumber)
 	crate_name = "food crate"
 
+/datum/supply_pack/organic/ice_cream
+	name = "Big Top Ice Cream Crate"
+	desc = "A crate with a carton of every flavor of Big Top brand ice cream."
+	cost = 2500
+	contains = list(/obj/item/storage/box/ice_cream_carton,
```

Comment:
you'll want to remove this from the list after you make it start empty

Reply:
It is crafted via activating cardboard which for some reason seems to bypass PopulateContents() because when I tested it, crafted ice cream cartons were empty.